nextcord/ext/commands/core.py

```diff
@@ -277,6 +277,17 @@ class Command(_BaseCommand, Generic[CogT, P, T]):
 
 
         .. versionadded:: 2.0
+    inherit_hooks: :class:`bool`, default=False
+        If ``True`` and this command has a parent :class:`Group` then this command
+        will inherit all checks, pre_invoke and after_invoke's defined on the the :class:`Group`
+
+        .. note::
+            You can define a ``pre_invoke`` or ``after_invoke`` on this
+            command in conjunction with ``inherit_hooks=True`` however
+            the one defined for this command will not be replaced by
```

Comment:
maybe rewrite it to say that the one you set per command will override the parent ones?